tardis/apps/publication_workflow/components/PublicationCard.jsx

```diff
@@ -0,0 +1,157 @@
+import PropTypes from 'prop-types';
+import React, { Fragment } from 'react';
+import styled from '@emotion/styled';
+import { Button } from 'react-bootstrap';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933947)
```suggestion
import { Button } from "react-bootstrap";
```